example-setup/scripts/install_deployhost.sh

```diff
@@ -27,12 +28,17 @@ cat <<EOF> /etc/consul.d/consul.json
   "ui": true,
   "bootstrap_expect": 3,
   "enable_script_checks": true,
+  "disable_remote_exec": true,
```

Comment:
I think we don't need to disable remote_exec if we have working ACLs for consul. Rather the opposite :)